src/install-otp.ps1

```diff
@@ -0,0 +1,19 @@
+param([Parameter(Mandatory=$true)][string]$VSN)
```

Comment:
All of this is basically a translation of `install-otp.sh`. I'm sure there's a better way to write a PowerShell script, but my knowledge here is very limited. I just wanted to make this work.